pom/product-selection-page.ts

```diff
@@ -0,0 +1,35 @@
+import puppeteer, { Puppeteer, type Locator, type Page } from "puppeteer-core";
+
+export class ProductSelectionPage {
+    readonly page: Page;
+    readonly radio_Leap: Locator<Element>
+    readonly radio_MicroOs: Locator<Element>
+    readonly radio_Tumbleweed: Locator<Element>
+    readonly selectButton: Locator<Element>;
+
+    constructor(page: puppeteer.Page) {
+        this.page = page;
+        this.radio_Leap = page.locator("::-p-text('Leap 16.0 Alpha')");
```

Comment:
this will change soon, from radio buttons to tiles I think, but no the point, at the end if the selector doesn't have any reference to radio button, the name of the variable to store shouldn't either.
we use only the text in this case, whatever the container of the text is, then would be Leap_text for example.
Another thing is that this is not Perl, so please watch out on the whole file for `_` as we should use camel style.
Additionally I'm not big fan of capital letter except for the camel notation, it makes me confused, same for the name of classes, it is much easier to always put 'tumbleweed' or 'leap' or even an acronyms in lower letter, it helps IMO.